packages/ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import { Slot } from '@radix-ui/react-slot'
+import { type RecipeVariants } from '@vanilla-extract/recipes'
+import clsx from 'clsx'
+import { type ComponentProps, type PropsWithChildren } from 'react'
+import { Heading, Text } from 'ui'
+import { cardHeader, cardRoot } from './Card.css'
+
+type RootStyleProps = RecipeVariants<typeof cardRoot>
+type RootProps = ComponentProps<'div'> & RootStyleProps
+
+function CardRoot({ variant, size, className, children }: PropsWithChildren<RootProps>) {
+  return <article className={clsx(cardRoot({ variant, size }), className)}>{children}</article>
+}
+
+type HeaderProps = ComponentProps<'header'>
+
+function CardHeader({ className, children }: PropsWithChildren<HeaderProps>) {
```

Comment:
nit:
I think this would be cleaner but I guess is personal preference so feel free to ignore this :)

```suggestion
type HeaderProps = PropsWithChildren<ComponentProps<'header'>>

function CardHeader({ className, children }: HeaderProps) {
```

Reply:
Sure! Looks good, will update!